src/main/java/org/tailormap/api/controller/GeoServiceProxyController.java

```diff
@@ -144,14 +144,14 @@ private void checkRequestValidity(
       throw new ResponseStatusException(HttpStatus.FORBIDDEN, "Proxy not enabled for requested service");
     }
 
-    if (authorisationService.mustDenyAccessForSecuredProxy(application, service)) {
+    if (authorisationService.mustDenyAccessForSecuredProxy(service)) {
       logger.warn(
```

Comment:
```suggestion
      logger.debug(
```
since this is no longer an indication of a misconfiguration I would decrease the severity of this message so it does not fill up the logs